src/common/__tests__/merjoonTransformer.test.ts

```diff
@@ -49,4 +56,44 @@ describe("Merjoon Transformer | parseTypedValue", () => {
     expect(type).toBeNull();
     expect(key).toBe('string("content")');
   });
+
+  it('Should return a number as value given a valid string in ISO format', () => {
+    const data = {
+      "created-on": "2024-05-08T18:07:33.852Z"
+    };
+    const path = 'Timestamp("created-on")';
+    const value = MerjoonTransformer.parseValue(data, path);
+
+    expect(value).toEqual(expect.any(Number))
```

Comment:
as we have the exact date string, let's assert for having equal to exact number instead of any number